tests/test_expressions.py

```diff
@@ -69,7 +77,7 @@ def test_hash(self):
             {
                 parse_one("select a.b"),
                 parse_one("1+2"),
-                parse_one('"a".b'),
+                parse_one('"a"."b"'),
```

Comment:
this shouldn't have broke, this was an intentional test

Reply:
Fixed.